x-pack/plugins/security_solution/common/utility_types.ts

```diff
@@ -40,8 +40,10 @@ export const unionWithNullType = <T extends runtimeTypes.Mixed>(type: T) =>
 export const stringEnum = <T>(enumObj: T, enumName = 'enum') =>
   new runtimeTypes.Type<T[keyof T], string>(
     enumName,
+    // @ts-expect-error upgrade typescript v4.9.5
     (u): u is T[keyof T] => Object.values(enumObj).includes(u),
     (u, c) =>
+      // @ts-expect-error upgrade typescript v4.9.5
```

Comment:
Hey @elastic/security-threat-hunting-investigations @janmonschke, it looks like the `stringEnum` function is used in the Timeline area, maybe you'd like to look into this change.

Reply:
Done in https://github.com/elastic/kibana/pull/175178/commits/177f8c99bb67142e95b4cb20ad68487c2f7c041d